### PR TITLE
GitHub Workflow for Publishing to Test-PyPi

### PR DESCRIPTION
**Pull request by Decarabia**
### Description:
This PR establishes a workflow for publishing packages to Test-PyPi. It can be expanded to publish to PyPi when we are ready.

### Potential Impact:

No potential impact.

### Unit Test Approach:

No unit tests.

### Test Results:

Package was successfully published.

Screen Shot 2020-03-30 at 1 57 26 PM

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[build-system]
+# These are the assumed default build requirements from pip:
+# https://pip.pypa.io/en/stable/reference/pip/#pep-517-and-518-support
+requires = ["setuptools>=40.8.0", "wheel"]
+build-backend = "setuptools.build_meta"
```
Comment (mattmanley) [inline pyproject.toml]: Nit: add extra line

**Comment by danielduhh** [comment]
For reference, the test version was posted: https://test.pypi.org/project/maproulette/0.0.1/

**Comment by danielduhh** [comment]
@Decarabia where does the workflow pick up the version number ?

**Comment by danielduhh** [comment]
Version is picked up - https://github.com/osmlab/maproulette-python-client/blob/dev/setup.py#L7

**Pull request merged**